content/doc/book/pipeline/syntax.adoc

```diff
@@ -105,16 +105,17 @@ section is placed. The section must be defined at the top-level inside the
 [[differences-between-top-and-stage-level]]
 ===== Differences between top and stage level Agents
 
-There are some nuances when adding an agent to the top level or a stage level, and this when the `options` directive is applied.
+There are some nuances when adding an agent to a top level or stage level, and this is when the `option` directive is applied.
+Refer to the link:../syntax/#options[options] section for more information.
 
 [[top-level-agents]]
 ====== Top Level Agents
-In `agents` declared at the outermost level of the Pipeline, the options are invoked *after* entering the agent.
-As an example, when using `timeout` it will be only applied to the execution *within* the agent.
+In `agents` declared at the outermost level of the Pipeline.
+First, an agent will be allocated to you, and then the option `timeout` will be triggered inside that agent.
```

Comment:
```suggestion
First, an agent is allocated to you, and then the option `timeout` is triggered inside that agent.
```
suggestion to make text more present, instead of future tense